install/preflight/aur-download.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+# Set up yay wrapper to use local PKGBUILDs if available
+echo "Setting up AUR package management..."
+
+if [[ -n "$OMARCHY_PKGBUILD_CACHE" ]] && [[ -d "$OMARCHY_PKGBUILD_CACHE" ]]; then
+    echo "✓ Using local PKGBUILD cache: $OMARCHY_PKGBUILD_CACHE"
+    
+    # Set up yay wrapper only if yay exists
+    if command -v yay &>/dev/null && [[ ! -f /tmp/yay-wrapped ]]; then
+        # Backup original yay
+        sudo cp /usr/bin/yay /usr/bin/yay.original 2>/dev/null || true
```

Comment:
Replacing yay like this seems like a bad idea?

Reply:
Agreed. This is a terrible idea.